tests/csapi/user_directory_display_names_test.go

```diff
@@ -0,0 +1,64 @@
+// +build !synapse_blacklist
+
+// Rationale for being included in Synapse's blacklist: https://github.com/matrix-org/synapse/issues/5677
```

Comment:
You probably _don't_ want this on Synapse's blacklist.

Reply:
My thinking was that I wouldn't want synapse CI to suddenly light up red. But maybe your point is "it should do!"